renovate.json

```diff
@@ -10,8 +10,18 @@
     },
     {
       "matchUpdateTypes": ["minor"],
+      "matchManagers": ["js"],
       "dependencyDashboardApproval": true
     },
+    {
+      "matchUpdateTypes": ["minor"],
+      "matchManagers": ["!js"],
+      "schedule": ["before 3am on Monday"]
+    },
+    {
+      "matchManagers": ["ci"],
```

Comment:
And I believe this one is 'github-actions', https://github.com/renovatebot/renovate/blob/main/lib/modules/manager/api.ts#L155